conjure-java-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/feignimpl/PathTemplateHeaderEnrichmentContract.java

```diff
@@ -19,10 +29,11 @@ public PathTemplateHeaderEnrichmentContract(Contract delegate) {
 
     @Override
     protected void processMetadata(Class<?> targetType, Method method, MethodMetadata metadata) {
+
         metadata.template().header(OkhttpTraceInterceptor.PATH_TEMPLATE_HEADER,
-                metadata.template().method() + " "
-                        + metadata.template().url()
-                            .replace('{', OPEN_BRACE_REPLACEMENT)
-                            .replace('}', CLOSE_BRACE_REPLACEMENT));
+                metadata.template().method() + " " + metadata.template().url()
+                        // escape from feign string interpolation
+                        // See RequestTemplate.expand
+                        .replace("{", "{{"));
```

Comment:
do we have any tests to verify that this doesn't break anything?

Reply:
Yes, this is tested by com.palantir.conjure.java.client.jaxrs.TracerTest

Reply:
:+1: I checked this out locally and verified that without that `.replace("{", "{{"));` line `com.palantir.conjure.java.client.jaxrs.TracerTest` fails!